### DISCUSSION TITLE
Could you provide cropped youtube-bb dataset?

### DISCUSSION THREAD

**Discussion by deneb2016** [STVIR/pysot#403] [General] [answered]
The original youtube-bb dataset is too large. (approximately 20TB?)
Could you provide cropped youtube-bb dataset?

Thanks in advance.

**Comment by laisimiao** [comment, marked as answer]
> The original youtube-bb dataset is too large. (approximately 20TB?)
> Could you provide cropped youtube-bb dataset?
> 
> Thanks in advance.

In [TRAIN.md](https://github.com/STVIR/pysot/blob/master/TRAIN.md),it has provided cropped youtube-bb dataset download link.
